tinker/static/assets/css/tinker.css

```diff
@@ -175,6 +180,11 @@ a#remove_new_job_button {
     border-color: #f04124 !important;
 }
 
+.danger {
```

Comment:
don't we already have .btn-danger with bootstrap? https://getbootstrap.com/docs/4.3/components/buttons/